_11ty/plugins/markdown-plugins.js

```diff
@@ -38,5 +39,10 @@ module.exports = function syntaxHighlighting(eleventyConfig) {
         uniqueSlugStartIndex: 1
     });
 
+    // add markdown containers
+    md = md
+        .use(require("markdown-it-container"), "correct", {})
+        .use(require("markdown-it-container"), "incorrect", {});
```

Comment:
```suggestion
        .use(markdownItContainer, "correct", {})
        .use(markdownItContainer, "incorrect", {});
```

To fix `'markdownItContainer' is assigned a value but never used`